src/nvim/edit.c

```diff
@@ -4171,6 +4171,22 @@ static int ins_compl_get_exp(pos_T *ini)
                            EW_FILE|EW_DIR|EW_ADDSLASH|EW_SILENT) == OK) {
         // May change home directory back to "~".
         tilde_replace(compl_pattern, num_matches, matches);
+#ifdef BACKSLASH_IN_FILENAME
+        if (curbuf->b_p_csl[0] != NUL) {
+          int i;
+          for (i = 0; i < num_matches; i++) {
+            char_u  *ptr = matches[i];
+            while (*ptr != NUL) {
+              if (curbuf->b_p_csl[0] == 's' && *ptr == '\\') {
+                *ptr = '/';
+              } else if (curbuf->b_p_csl[0] == 'b' && *ptr == '/') {
+                *ptr = '\\';
+              }
+              ptr += (*utfc_ptr2len)(ptr);
```

Comment:
I don't understand. This is an actual function, not a macro. Why wrap in `(* )`?

Reply:
Bad on my part. That was a blatant copy paste without thinking about the surrounding syntax.